packages/p2p/src/components/buy-sell/buy-sell-header.jsx

```diff
@@ -98,12 +98,20 @@ const BuySellHeader = ({ table_type }) => {
                         placeholder={isDesktop() ? localize('Search by nickname') : localize('Search')}
                     />
                     <SortDropdown />
-                    <Icon
-                        className='buy-sell__header-row--filter'
-                        icon='IcFilter'
-                        onClick={() => general_store.showModal({ key: 'FilterModal', props: {} })}
-                        size={40}
-                    />
+                    <div style={{ position: 'relative' }} className='buy-sell__header-row--filter'>
+                        <div
+                            className={classNames('buy-sell__header-row--filter-status', {
+                                'buy-sell__header-row--filter-status--disabled':
+                                    buy_sell_store.selected_payment_method_value.length === 0,
+                            })}
+                        />
+                        <Icon
+                            icon='IcFilter'
+                            className='buy-sell__header-row--filter-icon'
+                            onClick={() => general_store.showModal({ key: 'FilterModal', props: {} })}
+                            size={40}
+                        />
+                    </div>
```

Comment:
Added new status icon when user has selected some payment methods to filter:
<img width="456" alt="Screenshot 2023-06-22 at 4 43 26 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/9ed58ee8-ef37-448f-99c7-ec1bd7d726ef">

Reply:
POGGIES!! but it looks slightly bigger than the icon on the left

Reply:
Yeaa in the design it looks like that, if I make it smaller later QA will complain 🤣